plugins/zynaddsubfx/CMakeLists.txt

```diff
@@ -3,12 +3,12 @@ INCLUDE(BuildPlugin)
 
 # definitions for ZynAddSubFX
 IF(LMMS_BUILD_LINUX OR LMMS_BUILD_APPLE OR LMMS_BUILD_OPENBSD)
-    FIND_PACKAGE(X11)
-    INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
+        FIND_PACKAGE(X11)
+        INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
 	ADD_DEFINITIONS(-DOS_LINUX)
-ELSE()
```

Comment:
Hmmm.. I'm restarting the Travis job... this just doesn't make any sense.  We use this `cmake` syntax all over the place.